### ISSUE TITLE
Move community chat from Gitter to Discord

### ISSUE THREAD

**Issue by Piedone** [OrchardCMS/OrchardCore#13707]
### Is your feature request related to a problem? Please describe.
Gitter, what we use currently for community chat, is far from being the best chat app out there, particularly how it handles threads (or rather, how it's a feature that few people use, making the whole conversation hard to follow). [Discord](https://discord.com/) is a better chat app IMO, but more importantly that a subjective opinion, it seems that a lot of development communities use Discord but very few use Gitter (e.g. .NET itself, .NET Foundation, .NET Oxford, the unofficial C# Discord server). Discord has a cool factor.

### Describe the solution you'd like
I'd create a Discord server (which is free) and link that from the website, then close the Gitter channel. Discord can also have multiple channels, including voice ones, with various levels of moderators. We can eve add bots.

### Describe alternatives you've considered
My personal preference for technical Q&A is [the discussion board](https://github.com/OrchardCMS/OrchardCore/discussions). But apparently, many people prefer simpler chat.

**Comment by hishamco** [comment]
What about Slack?

**Comment by Skrypt** [comment]
Here is the link: https://discord.gg/hkgWCkGv
Expiring in 7 days.

**Comment by Piedone** [comment]
Hehh, I never knew this existed!

**Comment by agriffard** [comment]
I don't manage to access the Discord @Skrypt:

![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/e018a7d4-2304-41bd-96b0-2cb808280077)

**Comment by Piedone** [comment]
Here's a new link that doesn't expire: https://discord.gg/s3e2HtyPZc

**Comment by sebastienros** [comment]
@agriffard suggests adding a discord badge in the README

**Comment by Piedone** [comment]
So are we OK with moving from Gitter? Or anybody would like to defend it? (I don't see the point of having two chats.)

**Comment by BenedekFarkas** [comment]
Waaaaaaay better chat app, than Gitter, especially with threaded discussions enabled.

We could even host the weekly meeting there, although proper quality of screen sharing would require some server boosting at $35/month total value (that can come from server members too with Discord Nitro).

Happy to help with configuring the server!

**Comment by Piedone** [comment]
Perhaps we don't have higher usage because Gitter is not a good chat app? :) "Move", because we definitely don't want to have yet another official community support channel and keeping an eye on them.

**Comment by BenedekFarkas** [comment]
I would definitely be more inclined to be active on an Orchard Discord server (compared to Gitter), but I use Discord on almost a daily basis anyway.

**Comment by douwinga** [comment]
I am resurrecting this discussion to see if we can get any traction on this. Gitter is pretty much unusable on mobile these days as the menu covers most of the screen, and is not great on desktop compared to Discord. Also, I think most people are on Discord these days anyways, so it is easier to see activity as it is next to all the other communities people belong to. The OrchardCore Gitter used to be really active and was a great way to interact with other OrchardCore users for questions and ideas. I wonder if part of the reason for it not being very active anymore is due to people not liking Gitter for one reason or another.

**Comment by kevinchalet** [comment]
> Gitter is pretty much unusable on mobile these days as the menu covers most of the screen, and is not great on desktop compared to Discord.

FYI, Gitter is now based on Matrix. No idea if the Android app is similar, but Element on iOS - the flagship Matrix client - is fantastic.

**Comment by DrewBrasher** [comment]
I love using discord. I think it would be great if the community chat was moved there!

**Comment by douwinga** [comment]
> FYI, Gitter is now based on Matrix. No idea if the Android app is similar, but Element on iOS - the flagship Matrix client - is fantastic.

It was a bad experience when I tried. Years ago I was fairly active on gitter and it was pretty decent back then. The repos I used to follow have either died off or moved off of gitter, but I noticed OrchardCore was still on it so I tried it out to see if it was still active. The first thing I was greeted with is a menu covering the chat with no way to minimize the menu so I had to use landscape mode and still hardly see any chats. Then it prompted me to install the app. I was a little skeptical as the app it wanted me to install was not called Gitter, but Element which I had never heard of. I reluctantly installed the app, which then had me install a second Element app for some reason. It looked like the first app was just to have a link to the second app or something. After installing the second app I remember it not being too simple to find how to get on the OrchardCore channel, but I don't remember exactly why. Once I did get on the channel I had the same issue with needing to be in landscape. I then uninstalled both apps and searched for an OrchardCore discord channel. I found it, and although it isn't very active yet, that could change. @Piedone asked me to resurrect this discussion.

My experience could be similar to others, or people may give up before I even did which would lower engagement. This is just my perspective as somebody who is not a very active github user, but is trying to change that.

**Comment by Piedone** [comment]
@agriffard?

**Comment by agriffard** [comment]
Forum link replaced by Discord on OC homepage.

**Comment by Piedone** [comment]
Thank you!

**Comment by Piedone** [comment]
Nothing else really remains here.

**Comment by Piedone** [comment]
@OrchardCMS/orchardcore-devs I created a private channel on Discord just for those who have write access to the repo. If you haven't been added to it yet (because e.g. you haven't signed up to the Discord server yet), then please connect to the server (https://orchardcore.net/discord) and send me a PM there!

**Comment by Piedone** [comment]
@MikeAlhayek @sebastienros are you coming?

**Comment by MikeAlhayek** [comment]
Almost everyone is suggesting to move to Discord. I think we should move. Already see 70+ member that are there which is good. I think we should post a note on Gitter mentioning everyone and lock it/close it so that anyone visit it knows how to join us on Discord. Last thing our small community needs is division across multiple platforms. 

Also, for those who use our other channels, please try to be more active. Simple reaction or resharing posts will help spreading any news around. This will also draw attention to OrchardCore.

**Comment by Piedone** [comment]
Agree with both. I added a @room message on Gitter, what we unfortunately can't pin (https://gitlab.com/gitterHQ/webapp/-/issues/501). I also removed the remaining references here: https://github.com/OrchardCMS/OrchardCore/pull/16270. The only thing remaining is to remove the link from the website's header, but see https://github.com/OrchardCMS/OrchardCore.Website/issues/8.

**Comment by Piedone** [comment]
@kevinchalet you coming?

**Comment by MikeAlhayek** [comment]
Also @ns8482e

**Comment by kevinchalet** [comment]
> @kevinchalet you coming?

Actually, I joined [5 years ago](https://discord.com/channels/551136772243980291/551136772784914433/551148450641739799), but as @Skrypt said, it's never been super active to say the least 😃

**Comment by Piedone** [comment]
Now it is :). Are you Pinpoint there?
